IOS/SwiftSeedProject/SwiftSeedProject/Application/ViewModels/LoginViewModel.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  ViewModel.swift
+//  SwiftSeedProject
+//
+//  Created by Brian Sztamfater on 27/3/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Bond
+
+class LoginViewModel {
+    
+    // TODO: Create a service for authentication and remove references to Api
+    private let api: Api
+    private var user: User?
+    
+    var name = Observable<String?>("")
+    var email = Observable<String?>("")
+    
+    init() {
+        // TODO: We need to implement dependency injection
+        self.api = Api.init()
+    }
+    
+    func getRandomUser() {
```

Comment:
Minor.. but you have two options here

```
func getRandomUser() {
   //get random Id
   //retrieve user with random Id
}
```

or 

```
func getUser(userId: String) {
   //retrieve user with userId
}

getUser(String(arc4random_uniform(10) + 1));
```

The first one is limiting the re usability of the class, only suited to get random users, that's kind of lame 😛